Packs/DevSecOps/IncidentFields/DevSecOps_After_Commit_ID.json

```
@@ -0,0 +1,30 @@
+{
+    "id": "incident_aftercommitid",
+    "version": -1,
+    "modified": "2020-08-09T11:01:34.448790364+04:00",
+    "name": "After Commit ID",
+    "ownerOnly": false,
+    "cliName": "aftercommitid",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": [
+        "New Git PR"
```

```suggestion
        "Dev Sec Ops New Git PR"
```
Same in all incident fields/types indicator fields/types